test/lib/rules/yield-effects-spec.js

```diff
@@ -25,9 +79,73 @@ ruleTester.run("yield-effects", rule, {
   ],
   invalid: [
     {
+      code: "import { take } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = take();"
+          + "  yield sagaTakeEffect;"
+          + "}",
+      errors: [ERRORS.noyield]
+    },
+    {
+      code: "import { take as sagaTake } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = sagaTake();"
+          + "  yield sagaTakeEffect;"
```

Comment:
It's a valid piece of code, you can generally yield any value from a generator. The question is, do we allow that for sagas (the files where the `redux-saga` module is imported)?